packages/ui/src/primitives/tailwind/Button/index.tsx

```diff
@@ -73,34 +74,30 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
       rounded = 'partial',
       variant = 'primary',
       disabled = false,
-      className,
-      iconContainerClassName,
-      textContainerClassName,
       ...props
     },
     ref
   ) => {
     const twClassName = twMerge(
-      'flex items-center',
-      'font-medium text-white',
+      'flex items-center justify-center',
+      'text-sm font-medium leading-4 text-white',
+      'px-4 py-1',
       'transition ease-in-out',
       'disabled:cursor-not-allowed',
       (StartIcon || EndIcon) && 'justify-center',
       sizes[size],
       fullWidth ? 'w-full' : 'w-fit',
+      'min-w-[66px]',
       roundedTypes[rounded],
       disabled ? 'bg-[#F3F4F6] text-[#9CA3AF] dark:bg-[#5F7DBF] dark:text-[#FFFFFF]' : '',
```

Comment:
we don't need `disabled` prop. these classes can be added with `disabled:` modifier. And since `Button` props are inherited from HTML's buttons, disabled will still be available to Intellisense. And props will be passed to HTML's buttons through `{...props}`.

Reply:
Fixed